devops/dev_api.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 ./manage.py migrate --noinput
+./manage.py createinitialrevisions
```

Comment:
If running this is non-destructive and it doesn't take too long in the common case, I think this is a fine place.